### PR TITLE
Invite member from member page

### PR DESCRIPTION
**Pull request by yash6102000**

### CODE REVIEW

File: public/locales/en/index.json [hunk 1]
```diff
@@ -167,7 +176,10 @@
     "switched": "Workspace Switched",
     "enteredReenteredPassword": "Password and Confirm password should be same.",
     "passAlreadyCreated": "You have already created the password",
-    "userNotFound": "User Not Found"
+    "userNotFound": "User Not Found",
+    "workspaceJoinSuccess": "Workspace join successfully",
```
Comment (anuragP04) [inline public/locales/en/index.json:180]: Workspace joined successfully

File: app/models/invites.server.ts [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  const role = await prisma.role.findUnique({ where: { id: roleId } })
+  const workspaceJoinLink = env.PUBLIC_URL + '/workspace/' + user?.id + '/join'
+  return await sendMail(
+    user?.email as string,
+    user?.invitedForWorkspace?.name as string,
+    role?.name as string,
+    workspaceJoinLink as string
+  )
+}
+export async function reinviteMember({ id }: { id: string }) {
```
Comment (anuragP04) [inline app/models/invites.server.ts:128]: are we using this function?

File: app/models/invites.server.ts [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+      invitedById: {
+        select: {
+          firstName: true,
+          lastName: true,
+        },
+      },
+    },
+  })
+}
+export async function reInvitationMember({ id }: { id: string }) {
```
Comment (anuragP04) [inline app/models/invites.server.ts:113]: rename function, reinviteMemberForWorkspace

File: app/models/invites.server.ts [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+      },
+      invitedForWorkspace: {
+        select: {
+          name: true,
+        },
+      },
+    },
+  })
+}
+export async function deleteInviteMember(id: string) {
```
Comment (anuragP04) [inline app/models/invites.server.ts:78]: deleteMemberInvite, rename

File: app/components/workspace/JoinWorkspace.tsx [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+        <div className="flex max-w-554 flex-col items-center gap-10">
+          <img className="h-24 w-32" src={logo} alt="" />
+          <div className="flex flex-col gap-12">
+            <div className="flex flex-col items-center justify-center gap-4 text-2xl">
+              <span className="break-word text-2xl font-bold">
+                {' '}
+                {t('members.workspaceInvitation')}
+              </span>
+              {workspaceInvitation.joined === true ? (
+                <span className="text-center text-primary">
```
Comment (anuragP04) [inline app/components/workspace/JoinWorkspace.tsx:49]: use constants for all strings

File: app/models/sendgrid.servers.ts [hunk 6]
```diff
@@ -392,3 +392,88 @@ export async function sendNewPassword(email: string, password: string) {
...
+) {
+  const to = email
+  const from = 'careers@copods.co'
+  const subject = 'Welcome to K - Quiz @ Copods'
+  const text = 'K - Quiz @ Copods'
+  const logo = 'K - Quiz logo'
+  const html = `<html>
+  <head>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```
Comment (anuragP04) [inline app/models/sendgrid.servers.ts:409]: Screenshot 2022-11-17 at 10 29 30 AM

This  mail is not correctly placed in code, please check

File: public/locales/en/index.json [hunk 7]
```diff
@@ -8,7 +8,16 @@
...
     "member": "member",
-    "memberResendIcon": "Reinvite Icon"
+    "invitedMember": "Invited Member",
+    "joinedMembers": "Joined Members",
+    "noInvitedMember": "No any invited Member Found",
+    "workspaceInvitation": "Workspace Invitation",
+    "invitedOn": "Invited On",
+    "alreadyJoinedWorkspace": "You already joined the workspace",
+    "alreadyRejectedWorkspaceInvitation": "You already reject the workspace joining invitation",
+    "loggedinFromAnotherAccount": "You are currently logged in from another id, please login from your account first",
```
Comment (anuragP04) [inline public/locales/en/index.json:18]: Invalid link, please login from correct account.

**Comment by anuragP04** [comment]
when user joins the workspace, redirect him to dashboard by logging him to same workspace

File: app/components/login/SignUp.tsx [hunk 8]
```diff
@@ -69,7 +76,11 @@ const SignUp = ({ error }: { error?: string }) => {
   const onBlurConfPassError = () => {
     if (password !== confirmPassword && confirmPassword.length !== 0) {
       setOnConfBlurPasswordErr('settings.passNotMatch')
-    } else if (confirmPassword.length == 0 || password.length == 0) {
+    } else if (
+      confirmPassword.length == 0 ||
+      password.length == 0 ||
```
Comment (akshay-copods) [inline app/components/login/SignUp.tsx:81]: can use strict equality here

**Review by anuragP04 — APPROVED** [review]

**Pull request merged**